client/src/components/ProjectUsers.vue

```diff
@@ -0,0 +1,286 @@
+<script>
+import { mapState, mapActions, mapMutations } from 'vuex';
+import UserAvatar from '@/components/UserAvatar.vue';
+import djangoRest from '@/django';
+
+export default {
+  name: 'ProjectUsers',
+  components: {
+    UserAvatar,
+  },
+  inject: ['user'],
+  data: () => ({
+    showAddMemberOverlay: false,
+    showAddCollaboratorOverlay: false,
+    selectedPermissionSet: {},
+  }),
+  computed: {
+    ...mapState(['currentProject', 'allUsers']),
+    permissions() {
+      return this.currentProject.settings.permissions;
+    },
+    writePermission() {
+      return this.user.is_superuser;
+    },
+    members() {
+      const members = [...this.currentProject.settings.permissions.tier_1_reviewer];
+      members.push(...this.currentProject.settings.permissions.tier_2_reviewer);
+      return members;
+    },
+    collaborators() {
+      return this.currentProject.settings.permissions.collaborator;
+    },
+    changesMade() {
+      return JSON.stringify(this.permissions)
+        !== JSON.stringify(this.selectedPermissionSet);
+    },
+  },
+  watch: {
+    currentProject(newProj) {
+      this.selectedPermissionSet = { ...newProj.settings.permissions };
+    },
+  },
+  mounted() {
+    this.$store.dispatch('loadAllUsers');
+    this.selectedPermissionSet = { ...this.permissions };
+  },
+  methods: {
+    ...mapActions(['loadAllUsers']),
+    ...mapMutations(['setCurrentProject']),
+    getGroup(user) {
+      return Object.entries(this.permissions).filter(
+        ([, value]) => value.includes(user),
+      )[0][0].replace(/_/g, ' ');
+    },
+    async savePermissions() {
+      const newSettings = { ...this.currentProject.settings };
+      newSettings.permissions = Object.fromEntries(
+        Object.entries(this.selectedPermissionSet).map(
+          ([group, list]) => [group, list.map((user) => user.username || user)],
+        ),
+      );
+      try {
+        const resp = await djangoRest.setSettings(this.currentProject.id, newSettings);
+        this.showAddMemberOverlay = false;
+        this.showAddCollaboratorOverlay = false;
+        const changedProject = { ...this.currentProject };
+        changedProject.settings = resp;
```

Comment:
```suggestion
        changedProject.settings.permissions = resp.permissions;
```
Changing the user permissions discards any pending modifications to the import/export path.